modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/sql/IgniteSqlCreateIndex.java

```diff
@@ -41,6 +41,8 @@ public class IgniteSqlCreateIndex extends SqlCreate {
     /** Table name.  */
     private final SqlIdentifier tblName;
 
+    private final IgniteSqlIndexType type;
```

Comment:
Javadoc?

Reply:
According to coding guidelines, it's allowed to omit the javadoc for private members in case they are self-describing. I consider this is the case